azerothcore/azerothcore-wotlk
# fix(Core\Spells):Silithyst

## Changes Proposed:
-  SpellMgr requested change due to easier fix overriding effect 0 which provides immunity to.. nothing.
-  

## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/1018

## SOURCE:


## Tests Performed:

- Find a Silithyst Geyser or Silithyst Mound, click on it, notice you are not flagged for pvp, it does not reduces your speed and its not spawning a Silithyst Mound when you mount but gets removed when you mount.
- 


## How to Test the Changes:


1. Find a Silithyst Geyser or Silithyst Mound, click on it, notice you will now be flagged for pvp, it does reduce your speed and spawns a Silithyst Mound when you mount up.
2.
3.

## Known Issues and TODO List:




## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
